README.md

```diff
@@ -127,10 +127,17 @@ for abrv in doc._.abbreviations:
 ```
 ### EntityLinker
 
-The `EntityLinker` is a SpaCy component which performs linking to a knowledge base. Currently, the
-Unified Medical Language System and the Medical Subject Headings (MESH) are supported.
-The linker simply performs a string overlap search on named entities,
-comparing them with a knowledge base of 2.7 million concepts using an approximate nearest neighbours search.
+The `EntityLinker` is a SpaCy component which performs linking to a knowledge base. The linker simply performs
+a string overlap based search on named entities, comparing them with a knowledge base of 2.7 million concepts
```

Comment:
its a little unclear what the 2.7 number is referring to here because there are 5 options below. probably mention the 2.7 refers to the default, umls option.